src/client/activation/activationManager.ts

```diff
@@ -46,6 +53,9 @@ export class ExtensionActivationManager implements IExtensionActivationManager {
             this.docOpenedHandler = undefined;
         }
     }
+    public activateSync(): void {
```

Comment:
I'm not seeing where we call this method, do we still need it?

Reply:
Fixed